app/build.gradle

```diff
@@ -53,7 +53,7 @@ android {
             testCoverageEnabled true
 
             buildConfigField 'String', 'GOOGLE_ANALYTICS_TRACKER', '"UA-68882401-2"'
-            buildConfigField "String[]", "TRANSLATION_ARRAY", "new String[]{}"
+            buildConfigField "String[]", "TRANSLATION_ARRAY", getAllLocales()
```

Comment:
Please don't do this. We disabled it for debug build to speed up development process. If you need please keep this change local.